### PR TITLE
Feature/config with cli

### PR DESCRIPTION
**Pull request by Joinhack**

### CODE REVIEW

File: blockless-cli/src/cli_clap.rs [hunk 1]
```diff
@@ -1,68 +1,117 @@
...
+use clap::{Arg, ArgMatches, Command, Parser};
+use url::Url;
+
+use crate::config::CliConfig;
+
+const INPUT_HELP: &str  = 
+    "the input file is wasm file, configure file, or car file";
+
+const DEBUG_INFO_HELP: &str  = 
+    "the input file is wasm file, configure file, or car file";
```
Comment (Maelkum) [inline blockless-cli/src/cli_clap.rs:15]: Should be `the debug info for the runtime` or something like that?

File: blockless-cli/src/cli_clap.rs [hunk 2]
```diff
@@ -1,68 +1,117 @@
...
+    #[clap(long = "runtime-logger", value_name = "RUNTIME-LOGGER", help = RUNTIME_LOGGER_HELP)]
+    runtime_logger: Option<String>,
+
+    #[clap(long = "limited-memory", value_name = "LIMITED-MEMORY", help = LIMITED_MEMORY_HELP)]
+    limited_memory: Option<u64>,
+
+    #[clap(long = "run-time", value_name = "RUN-TIME", help = RUN_TIME_HELP)]
+    run_time: Option<u64>,
+
+    #[clap(long = "entry", value_name = "ENTERY-HELP", help = ENTRY_HELP)]
```
Comment (Maelkum) [inline blockless-cli/src/cli_clap.rs:80]:
```suggestion
    #[clap(long = "entry", value_name = "ENTRY-HELP", help = ENTRY_HELP)]
```

File: blockless-cli/src/cli_clap.rs [hunk 3]
```diff
@@ -1,68 +1,117 @@
...
+    entry: Option<String>,
+
+    #[clap(long = "limited-fuel", value_name = "ENTERY-HELP", help = LIMITED_FUEL_HELP)]
```
Comment (Maelkum) [inline blockless-cli/src/cli_clap.rs:83]: `value_name` is `ENTERY-HELP`, should be `LIMITED-FUEL-HELP` or something like that?

File: blockless-cli/src/cli_clap.rs (hunk 1 above)
Comment (Joinhack) [reply blockless-cli/src/cli_clap.rs:15]: yes you are right

File: blockless-cli/src/cli_clap.rs (hunk 2 above)
Comment (Joinhack) [reply blockless-cli/src/cli_clap.rs:93]: this is done.

File: blockless-cli/src/cli_clap.rs (hunk 3 above)
Comment (Joinhack) [reply blockless-cli/src/cli_clap.rs:96]: done

Comment (Joinhack) [reply blockless-cli/src/cli_clap.rs]: thank you for you feedback.

**Review by dmikey — APPROVED** [review]

**Pull request merged**